### ISSUE TITLE
Accessibility of workspace

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1113]
Various issues with accessibility of workspace:

- [x] There's no alt text on Xerte or Apereo logos on header or any of images on footer bar
- [x] Rename logout to log out (pronounced very oddly in screenreader!)
- [x] 'Sort' string needs to be marked up as label for the combo box
- [x] The language selector label tag is empty
- [x] Use headings, regions and roles on the different areas
- [x] Lots of links that all say 'See example' - these should be different, e.g. 'Open XOT example', 'Open Bootstrap example'
- [x] No labels on inputs where you enter a name for project you want to create
